Assets/Scripts/MIDI2Event/MTrkTypes/MTrkEvent.cs

```diff
@@ -5,7 +5,7 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace midi2event
+namespace MIDI2EventSystem
 {
     internal class MTrkEvent
```

Comment:
maybe the property in this class should be DeltaTime instead of Delta to make that clearer (if I am understanding the webpage I pulled up about MTrk correctly--if it's something else, leave a comment saying what)

Reply:
The reason I am using delta instead of deltatime is because this value is not in units of time directly, but needs to be converted into units of time using other known constants, and in the context of game engines, deltatime is usually for values directly in seconds.